arbitrum-docs/node-running/node-providers.mdx

```diff
@@ -1,29 +1,49 @@
-# RPC providers / Node infrastructure
+---
+title: "RPC endpoints and providers"
+sidebar_label: "RPC endpoints and providers"
+description: Find available RPC endpoints and providers in the ecosystem
+reader_audience: developers who want to build on Arbitrum
+content_type: overview
+---
 
 :::info WANT TO BE LISTED HERE?
 
 Complete [this form](https://docs.google.com/forms/d/e/1FAIpQLSdw0U-9LcLuih5TZ_QghS-S_MS4wCKSEigA_IQEza_hFmNVow/viewform) if you'd like to see your project added to this list (and the [Arbitrum portal](https://portal.arbitrum.one/)). 
 
 :::
 
-To interact with public Arbitrum chains, you can rely on many of the same popular node providers that you are already using on Ethereum:
+## RPC endpoints
+
+This section provides an overview of the available public RPC endpoints for different Arbitrum chains and necessary details to interact with them.
+
+| Name                      | RPC Url(s)                            | Chain ID | Block explorer             | Underlying L1 | Tech stack | Sequencer feed URL                   |
+| ------------------------- | ------------------------------------- | -------- | -------------------------- | ------------- | ---------- | ------------------------------------ |
+| Arbitrum One              | https://arb1.arbitrum.io/rpc          | 42161    | https://arbiscan.io/       | Ethereum      | Nitro      | wss://arb1.arbitrum.io/feed          |
+| Arbitrum Nova             | https://nova.arbitrum.io/rpc          | 42170    | https://nova.arbiscan.io/  | Ethereum      | AnyTrust   | wss://nova.arbitrum.io/feed          |
+| Arbitrum Goerli (Testnet) | https://goerli-rollup.arbitrum.io/rpc | 421613   | https://goerli.arbiscan.io | Goerli        | Nitro      | wss://goerli-rollup.arbitrum.io/feed |
+
+🔉 Note: You can also find additional RPC endpoints available for Arbitrum chains in Chain Connect: [Arbitrum One](https://www.alchemy.com/chain-connect/chain/arbitrum-one), [Arbitrum Nova](https://www.alchemy.com/chain-connect/chain/arbitrum-nova) and [Arbitrum Goerli](https://www.alchemy.com/chain-connect/chain/arbitrum-goerli).
```

Comment:
(not a high priority, but I've been replacing these `Notes` with admonitions wherever I catch them)

```suggestion

:::info More RPC endpoints

More Arbitrum chain RPC endpoints can be found in Chain Connect: [Arbitrum One](https://www.alchemy.com/chain-connect/chain/arbitrum-one), [Arbitrum Nova](https://www.alchemy.com/chain-connect/chain/arbitrum-nova) and [Arbitrum Goerli](https://www.alchemy.com/chain-connect/chain/arbitrum-goerli).

:::

```